test/standalone/Vegetation/canopy_model.jl

```diff
@@ -551,277 +494,6 @@ end
     end
 end
 
-@testset "Canopy software pipes with energy model" begin
```

Comment:
similar here - can we keep the test, but remove the loops and just test in one config?

Reply:
oh I see, we are testing the energy model, above now?

Reply:
Yeah, since this and the "canopy software pipes" test above are so similar (only testing different energy models), I just modified the one above to use BigLeaf energy and added the corresponding check there